tencentcloud/resource_tc_cam_role.go

```diff
@@ -441,6 +453,10 @@ func resourceTencentCloudCamRoleUpdate(d *schema.ResourceData, meta interface{})
 			return err
 		}
 	}
+
+	if d.HasChange("session_duration") {
+		return fmt.Errorf("`session_duration` do not support change now.")
```

Comment:
This will cause terraform apply to fail after first time. because session_duration has not been read yet, it's always null.